modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/drone.dm

```diff
@@ -0,0 +1,65 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat/drone
+	name = "alien drone"
+	caste = "drone"
+	maxHealth = 200
+	health = 200
+	icon_state = "aliendrone"
+	/// Holds the healing aura ability the drone will be granted
+	var/datum/action/cooldown/alien/skyrat/heal_aura/heal_aura_ability
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	next_evolution = /mob/living/carbon/alien/humanoid/skyrat/praetorian
+
+/mob/living/carbon/alien/humanoid/skyrat/drone/Initialize(mapload)
+	. = ..()
+	heal_aura_ability = new /datum/action/cooldown/alien/skyrat/heal_aura()
+	heal_aura_ability.Grant(src)
+
+/mob/living/carbon/alien/humanoid/skyrat/drone/Destroy()
+	QDEL_NULL(heal_aura_ability)
+	return ..()
+
+/mob/living/carbon/alien/humanoid/skyrat/drone/create_internal_organs()
+	internal_organs += new /obj/item/organ/internal/alien/plasmavessel
+	internal_organs += new /obj/item/organ/internal/alien/resinspinner
+	..()
```

Comment:
This is kinda ugly if that's really how that works.

Reply:
that's how mafia works